src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const values = Object.values(action);
```

Comment:
I think it's unnecessary step. 
You can easily get the type of action using `action.type` in your conditions